docs/dev/framework/security.md

```diff
@@ -0,0 +1,275 @@
+---
+title: "Security"
+description: Contao's Symfony Security implementation.
+aliases:
+    - /framework/security/
+---
+
+Contao uses [Symfony's Security Component][SymfonySecurityComponent] to handle front end and back end user authentication and authorization.
+The Contao Managed Edition provides its own [firewall][SymfonyFirewall], [user providers][SymfonyUserProvider] and 
+[access control][SymfonyAccessControl] via the `contao/manager-bundle`. If you do not use the Managed Edition and added Contao to your
+custom Symfony application, you will have to register Contao's security settings yourself, as mentioned in the 
+[Getting Started article][ContaoConfiguration]. Contao's Symfony Security configuration [in Contao **4.9**][Contao49SecurityYaml] looks like this for example:
```

Comment:
not sure it's a good idea to duplicate this here again. Maybe it should be referenced to the actual file instead?

Reply:
It already links to the actual file ;). We duplicate other things as well (like the config reference) so I think it's fine to have this here, to give a quick overview.